broken-site-reporting/README.md

```diff
@@ -53,4 +60,15 @@ for $testSet in test.json
     if $test.expectReportURLParams
         for $param in $test.expectReportURLParams
             expect($url.matchesRegex(/[?&] + $param.name + '=' + $param.value + [&$]/))
-```
\ No newline at end of file
+```
+
+## Platform exceptions
+
+Please see https://app.asana.com/0/1198207348643509/1200191185434196/f for
+current platform support and further links. A summary of exceptions affecting
+the reference tests is provided below:
+
+- description tests:
+  - the following platforms do not currently allow users to provide a description: `android-browser`, `ios-browser`, `safari-extension`
+- truncation tests:
+  - the following platforms do not currently implement the optional `noActionRequests`, `adAttributionRequests`, `ignoredByUserRequests`, or `ignoreRequests` truncatable parameters: `android-browser`, `ios-browser`, `macos-browser`, `safari-extension`, `windows-browser`
```

Comment:
something seems wrong here - you have both optional params and turntable params in one sentence, I think you meant to split it somehow?

Reply:
@kdzwinel see the documentation - all the truncatable parameters are currently optional. The requirement is that if a platform were to implement one (or more) of those optional parameters, then they must also implement truncation.